From c11008008631bd3428e5c413ea461b4b132274c8 Mon Sep 17 00:00:00 2001
From: Ewan Mellor <ewan.mellor@citrix.com>
Date: Wed, 9 Nov 2011 23:24:25 -0800
Subject: [PATCH] nova-manage: exit with status 1 if an image registration
 fails

This indicates the failure to the user through the exit status.

Original author: Renuka Apte <renuka.apte@citrix.com>

(cherry picked from commit d7e5a791503b67c53bd497b9cdebb19083630b14)

Change-Id: I73fc2cc35530e7ee7f113f5aaec7b577663e4dc1
---
 bin/nova-manage |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/bin/nova-manage b/bin/nova-manage
index 8a16202..3ff73f1 100755
--- a/bin/nova-manage
+++ b/bin/nova-manage
@@ -1810,6 +1810,7 @@ class ImageCommands(object):
             return new
         except Exception as exc:
             print _("Failed to register %(path)s: %(exc)s") % locals()
+            sys.exit(1)
 
     @args('--image', dest='image', metavar='<image>', help='Image')
     @args('--kernel', dest='kernel', metavar='<kernel>', help='Kernel')
-- 
1.7.6.5

